### DISCUSSION TITLE
Verzeichnis für eigene patch-Files

### DISCUSSION THREAD

**Discussion by Weseler485** [OpenCCU/OpenCCU#1766] [🇩🇪 Fragen&Antworten]
Hallo,
gibt es ein Verzeichnis auf dem System, wo ich eigene Scripte ablegen kann die nach einem Update a) noch vorhanden sind und b) genau ein einziges mal ausgeführt werden?

Ich habe eigene patch-Files die ich nach jedem Update wieder importieren möchte.

**Comment by Baxxy13** [comment]
zu a:
Alles was irgendwo im /usr - Verzeichnis liegt, bleibt von Zentralen-Updates unberührt

zu b:
dafür musst du selbst sorgen, z.B. mit einer "Kontroll-Datei" irgendwo außerhalb des /usr -Verzeichnisses. 
Legst du bspw. irgendwas in /www ab ist das nach dem Zentralen-Update wieder weg.
Dein Patch-Script lässt du nur ausführen wenn die Kontroll-Datei nicht vorhanden ist, nach dem patchen legst du die dann an damit beim Reboot nicht wieder gepatcht wird.
